Formula/vice.rb

```diff
@@ -39,20 +39,19 @@ def install
       "--prefix=#{prefix}",
       "--disable-dependency-tracking",
       "--disable-arch",
-      "--disable-bundle",
       "--enable-external-ffmpeg",
     ]
 
     if build.head?
       configure_flags << "--enable-native-gtk3ui"
-      configure_flags << "--disable-hwscale"
     else
+      configure_flags << "--disable-bundle"
```

Comment:
Please explain the reasons behind the changes.

Reply:
--disable-bundle was required in vice 3.3 to allow a 'make install' - required for brew. The HEAD codebase no longer requires or supports this flag.

--disable-hwscale was previously required for macOS gtk HEAD builds as openGL wasn't being used correctly on macOS.

Reply:
ok, that change is made.